### ISSUE TITLE
Can't output to network share

### ISSUE THREAD

**Issue by blueal** [yaser01/mkv-muxing-batch-gui#90]
Attempting to set output directory to a SMB share using an address similar to: `\\server.mydomain.com\Media` results in the following error:
```
Invalid Path
\\server.mydomain.com\Media isn't a valid path!
``` 
![image](https://github.com/yaser01/mkv-muxing-batch-gui/assets/4614311/586d46a4-1f1f-49be-aa58-fb91705330f2)

**Comment by yaser01** [comment]
Thanks for notifying me 
Will fix it ASAP

**Comment by yaser01** [comment]
Fixed in 2.4 Update :D
